pkg/ui/tailwind.config.cjs

```diff
@@ -3,5 +3,19 @@ const theme = require('../scaffold/tailwind.theme');
 module.exports = {
 	content: ['./src/**/*.{html,js,svelte,ts}'],
 	theme,
-	plugins: [require('@tailwindcss/forms')]
+	plugins: [
+		require('@tailwindcss/forms'),
+		function ({ addUtilities }) {
+			addUtilities({
+				'.scrollbar-hide::-webkit-scrollbar': {
+					display: 'none'
+				},
+
+				'.scrollbar-hide': {
+					'-ms-overflow-style': 'none',
+					'scrollbar-width': 'none'
+				}
```

Comment:
Could have used nesting (instead of having `.scrollbar-hide::--webkit-scrollbar` could have simply used `&::--webkit-scrollbar` inside of `.scrollbar-hide`)